x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/transforms/SettingsConfig.java

```diff
@@ -98,9 +98,9 @@ public Integer getDatesAsEpochMillisForUpdate() {
 
     public ActionRequestValidationException validate(ActionRequestValidationException validationException) {
         // TODO: make this dependent on search.max_buckets
-        if (maxPageSearchSize != null && (maxPageSearchSize < 10 || maxPageSearchSize > 10_000)) {
+        if (maxPageSearchSize != null && (maxPageSearchSize < 10 || maxPageSearchSize > 65_536)) {
             validationException = addValidationError(
-                "settings.max_page_search_size [" + maxPageSearchSize + "] must be greater than 10 and less than 10,000",
+                "settings.max_page_search_size [" + maxPageSearchSize + "] must be greater than 10 and less than 65,536",
```

Comment:
you found a bug, this error message is mathematically not correct, because 10 and `10,000` are allowed values. I am checking this and might come back with an improved wording.

Reply:
Can you use this error message:

```
"settings.max_page_search_size [" + maxPageSearchSize + "] is out of range. The minimum value is 10 and the maximum is " + MultiBucketConsumerService.DEFAULT_MAX_BUCKETS
```